modules/packages/ExternalArray.chpl

```diff
@@ -25,6 +25,18 @@
 module ExternalArray {
   use ChapelStandard;
 
+  extern type free_func;
+  private extern const FREE_FUNC_NIL: free_func;
+  private extern const FREE_FUNC_CHAPEL_WRAP: free_func;
+
+  extern record external_array {
+    var elts: c_void_ptr;
+    var size: uint;
+
+    var freer: free_func;
```

Comment:
Since you don't use it, I wouldn't even put freer in this record, but you can if you want.